ocs_ci/templates/workloads/fio/cr_template.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
`cr_template.yaml` is too generic name IMO

Reply:
This is now changed to `benchmark_fio.yaml`